### PR TITLE
[PIE-2215] Switch to sentence case for headings

### PR DESCRIPTION
**Pull request by sgregglives**
Signed-off-by: Steven Gregg <steven.gregg@consensys.net>

## PR description
Change from headline to sentence capitalization in headings
## Fixed Issue(s)

PIE-2215

### CODE REVIEW

File: docs/HowTo/Quickstart.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Start Orion specifying the [configuration file](#3-create-a-configuration-file):
 orion orion.conf
 ```
 
-### 5. Confirm Orion is Running
+### 5. Confirm Orion is funning
```
Comment (MadelineMurray) [inline docs/HowTo/Quickstart.md]: Typo

File: docs/Reference/Orion-CLI-Syntax.md [hunk 2]
```diff
@@ -41,8 +41,7 @@ Displays help and exits.
 
 Displays version information and exits.
 
- 
-## Configuration File 
+### configuration file
```
Comment (MadelineMurray) [inline docs/Reference/Orion-CLI-Syntax.md]: Is this lower-case c deliberately?  And the change in heading level?

File: docs/HowTo/Quickstart.md (hunk 1 above)
Comment (sgregglives) [reply docs/HowTo/Quickstart.md]: Fixed.

File: docs/Reference/Orion-CLI-Syntax.md (hunk 2 above)
Comment (sgregglives) [reply docs/Reference/Orion-CLI-Syntax.md]: No. Fixed now.

**Review by MadelineMurray — APPROVED** [review]

**Pull request merged**